setup.py

```diff
@@ -21,6 +22,10 @@ def read(fname):
     if not requirement.lstrip().startswith('#')
 ))
 
+
+if platform.system() == 'Windows':
+    install_requirements.append('python-certifi-win32==1.2')
```

Comment:
I wonder if you could just add the following line in `requirements.txt`

```
python-certifi-win32==1.2; platform_system=="Windows"
```

As per the [Environment Markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) syntax. Note, I haven't tested this, but expect it to work.

Reply:
Would probably work. But the problem is not this. It's that this package did not solve the problem for the user who reported it (or we did a mistake at packaging).